provider/resource_harbor_project.go

```diff
@@ -137,8 +137,9 @@ func resourceProjectDelete(d *schema.ResourceData, meta interface{}) error {
 	}
 
 	charts, err := client.GetCharts(projectName)
-	if err != nil {
-		return handleNotFoundError(err, d)
+	// this can return a 404 if chartmuseum is disabled
+	if err != nil && !harbor.ErrorIs404(err) {
```

Comment:
this is the actual change.  testing it requires a harbor deployment with chartmuseum disabled